pytorch_blade/torch_blade/pass_manager.py

```diff
@@ -308,11 +309,15 @@ def remove_raise_exception(outer_block):
 
 
 def _optimize_common(c_module, static_shape=False):
-    _jit_pass_quantization_preprocess(c_module)
+
     is_training = c_module.hasattr("training") and c_module.training
+    cfg = Config.get_current_context_or_new()
+    if is_training and cfg.enable_int8:
+        logger.error("If do quantization, the model must in eval mode ")
+    _jit_pass_quantization_preprocess(c_module)
```

Comment:
Nit: call this pass only when `enable_in8`:
```
if cfg.enable_int8：
    _jit_pass_quantization_preprocess(c_module)

Reply:
done